modules/installation/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -1,6 +1,6 @@
 [id='proc-rhdh-deploy-eks-using-operator_{context}']
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+= Installing {product} on {eks-name} ({eks-short}) using the Operator
```

Comment:
Why are we replacing the word 'deploy' with 'install'? Installing usually refers to the initial setup phase. Deployment, however, is a more comprehensive term that includes all the activities needed to make software or systems available and operational in their intended environment.

The procedure in this section covers all those activities and steps for the user. I would recommend retaining the term 'deploy' instead of substituting it with 'install.

Reply:
This is a great question, and one that has been asked in multiple forums. Several people have said that these terms are interchangeable and/or that "install" is acceptable for this use case. However, I continue to ask about and to welcome opinions on the "install" vs. "deploy" terminology topic.

Reply:
I would agree that we're installing the operator or chart, then deploying 1 or more instances of RHDH. Chart and operator are just the installation mechanism; creating an instance is the deployment step.